src/test/java/de/berlin/lostberlin/service/BusinessServiceTest.java

```diff
@@ -63,13 +77,8 @@ public void retrieveBusinessByLocationTest() {
 
         List<BusinessShortDao> allBusinesses = singletonList(businessShortDao);
 
-        given(businessRepository.getShortBusinessProfiles(location)).willReturn(allBusinesses);
-
         List<BusinessShortDao> business = businessService.retrieveBusinessByLocation(location);
 
```

Comment:
This test fails because there is no test data in the repo. For this purpose we can use EntityManager to directly place a test data to db and then use businessService method to check whether it is properly retrieved.